examples/code-http-anthropic/n8n-kit/.bundle-py/INzyo95PUum4xEuiL9MT/generate-prompt.py

```
@@ -0,0 +1,15 @@
+def handler(input):
+    if(input["type"] == "type1"):
+        return {
+            "prompt": f"Reply as if you were a gangster."
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)